q2_feature_classifier/_cutter.py

```diff
@@ -170,8 +170,10 @@ def extract_reads(sequences: DNASequencesDirectoryFormat, f_primer: str,
     """
     if min_length > trunc_len - trim_left and trunc_len > 0:
         raise ValueError('Your minimum length is larger than your truncation '
-                         'length minus your trim, this will result in all of '
-                         'your sequences being trimmed automatically.')
+                         'length minus your trim. This will result in all of '
+                         'your sequences being removed from your dataset. To '
+                         'procees, you must set a minimum length less than '
+                         'your truncation length minus your trim.')
```

Comment:
the wording is a bit awkward on this error message now. How about this:
"The minimum length setting is greater than the length of the truncated sequences. This will cause all sequences to be removed from the dataset. To proceed, set a min_length ≤ trunc_len - trim_left."